composer.json

```diff
@@ -18,7 +18,7 @@
     "mollie/mollie-api-php": "^2.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "^5 || ^6.5 || ^7.1",
+    "phpunit/phpunit": "^5 || ^7.1",
```

Comment:
Since the phpunit xml has been changed by not throwing exceptions on notices and warnings, do we still need to remove the 6.5 branch here?

Reply:
yeah, because for some reason I don't understand, the changes in phpunit.xml.dist did not prevent travis from using phpunit 6.5.14 when php 7.0 and it was throwing the warning as error anyway. I found this was the only way to make it work...any suggestions accepted

Reply:
Ok, I thought that preventing to fail when `failOnWarning` were false could solve the problem.

Btw, ok for me this solutions since php 7.0 works with phpunit 5.7.

Well done btw.